### PR TITLE
Added draft-07 schemas and examples

### PR DESCRIPTION
**Pull request by kunagpal**
1. This also updates the general project structure to make further additions easier.
2. All existing URLs remain unchanged and functional.
3. New draft schemas will be available under https://schemas.getpostman.com/json/draft-07/collection/<version\>/collection.json

### CODE REVIEW

File: schemas/draft-07/v1.0.0/collection/current-helper.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "$id": "#/definitions/current-helper",
+    "title": "Current auth helper",
+    "oneOf": [
```
Comment (codenirvana) [inline schemas/draft-07/v1.0.0/collection/current-helper.json]: `"type": ["string", "null"]`

File: schemas/draft-07/v2.0.0/collection/cookie.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    "title": "Cookie",
+    "$id": "#/definitions/cookie",
+    "description": "A Cookie, that follows the [Google Chrome format](https://developer.chrome.com/extensions/cookies)",
+    "properties": {
+        "domain": {
+            "type": "string",
+            "description": "The domain for which this cookie is valid."
+        },
+        "expires": {
+            "oneOf": [
```
Comment (codenirvana) [inline schemas/draft-07/v2.0.0/collection/cookie.json]: `"type": ["string", "number"]`

File: schemas/draft-07/v2.1.0/collection/cookie.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    "title": "Cookie",
+    "$id": "#/definitions/cookie",
+    "description": "A Cookie, that follows the [Google Chrome format](https://developer.chrome.com/extensions/cookies)",
+    "properties": {
+        "domain": {
+            "type": "string",
+            "description": "The domain for which this cookie is valid."
+        },
+        "expires": {
+            "oneOf": [
```
Comment (codenirvana) [inline schemas/draft-07/v2.1.0/collection/cookie.json]: `"type": ["string", "number"]`

**Review by codenirvana — CHANGES_REQUESTED** [review]
LGTM 👍 

Found a few places where `oneOf` can be replaced with the type shorthand.

**Comment by codenirvana** [comment]
Adding `TODO` for future PRs:
- [ ] Have directory structure like: `draft-<version>/<category>/<category-version>/` to accommodate  different categories of schemas.
- [ ] Format all JSON files to have consistent indentation (2 spaces).
- [x] `$schema` key is missing from some reference schemas.

File: examples/draft-04/v2.0.0-draft.4/box.json [hunk 4]
```diff
@@ -4,7 +4,7 @@
         "name": "Box Content API",
         "_postman_id": "48e1ee04-a604-2549-9422-43270e936399",
         "description": "I am trying to update this.",
-        "schema": "http://schema.getpostman.com/collection/v2/collection.json"
+        "schema": "http://schema.getpostman.com/json/collection/v2.0.0-draft.4/collection.json"
```
Comment (codenirvana) [inline examples/draft-04/v2.0.0-draft.4/box.json:7]: Shouldn't this be like:
`https://schemas.getpostman.com/json/draft-<version>/collection/<version>/collection.json`?

Comment (kunagpal) [reply examples/draft-04/v2.0.0-draft.4/box.json:7]: No. `draft4` links cannot be changed, they could break a lot of things.

**Review by codenirvana — APPROVED** [review]

**Pull request merged**